tests/BladeTest.php

```diff
@@ -142,6 +142,16 @@ public function the_hasanyrole_directive_will_evaluate_true_when_the_logged_in_u
     }
 
     /** @test */
+    public function the_hasanyrole_directive_will_evaluate_true_when_the_logged_in_user_does_have_some_of_the_required_roles_use_pipe_for_the_given_guard()
```

Comment:
This method name is a bit too long for my taste. What do you think about losing the `use_pipe_for_the_given_guard` part? (same remarks for the other test you added)